app/src/main/java/chat/rocket/android/chatroom/ui/ChatRoomFragment.kt

```diff
@@ -476,11 +480,17 @@ class ChatRoomFragment : Fragment(), ChatRoomView, EmojiKeyboardListener, EmojiR
     override fun sendMessage(text: String) {
         ui {
             if (!text.isBlank()) {
-                if (!text.startsWith("/")) {
-                    presenter.sendMessage(chatRoomId, text, editingMessageId)
-                } else {
+                if (text.startsWith("/")) {
                     presenter.runCommand(text, chatRoomId)
+                    return@ui
+                } else if (text.startsWith("+") && presenter.getLastMessageId() != null) {
+                    val trimmedText = text.substring(1).trimEnd()
```

Comment:
Couldn't we move all these stuffs (trimming, etc) to the presenter?

Reply:
Sure, moved it. Please review changes.